test/system/080-pause.bats

```diff
@@ -106,7 +106,7 @@ load helpers.systemd
     run_podman healthcheck run $ctrname
     is "$output" "" "output from 'podman healthcheck run'"
 
-    run -0 systemctl status $cid-*.{service,timer}
+    run -0 systemctl status $cid-*.timer
```

Comment:
Make sure to add a comment why we can only check the timer and link to the issue so readers know why this is only checking one unit